ansible/playbook.yaml

```
@@ -1,9 +1,45 @@
+        msg: 'The REGISTRY_PASSWORD or REGISTRY_USERNAME environment variable is not set. Please set it before running the playbook.'
+      when: lookup('env', 'REGISTRY_PASSWORD') == '' or lookup('env', 'REGISTRY_USERNAME') == ''
+
+    - name: Set registry password from environment variable
+      set_fact:
+        registry_username: "{{ lookup('env', 'REGISTRY_USERNAME') }}"
+        registry_password: "{{ lookup('env', 'REGISTRY_PASSWORD') }}"
+      tags:
+        - registry
+        - registry-auth
+        - registry-htpasswd
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)